<p><strong>Tree format</strong></p>
<ul>
	<li>Trees handled by CompPhy are represented in <i>Newick format</i>. A detailed presentation of this format can be read <a href="http://evolution.genetics.washington.edu/phylip/newicktree.html">here</a>. This format can be obtained in output of many tree inference software/webstes, such as Phylip, Paup*, Mega, PhyML, RaxML, or <a href="http://www.phylogeny.fr">Phylogeny.fr</a>.
<p>There are other format to describe trees, but for a start, we focused on the Newick format. So if you happen to have trees in another format such as Nexus or NHX, before uploading them to CompPhy you first have to convert them into Newick format by using conversion programs (e.g. try <a href="http://iubio.bio.indiana.edu/soft/molbio/readseq/">Readseq</a> if your trees are in Nexus format).
</li>
</ul>

<p><strong>Used software</strong></p>
<ul>
	<li><a href="http://www.scriptree.org">ScripTree</a>: used to generate the trees. Check its website for more information about its syntax etc. May be needed for the manual tuning of the picture functionality.

	<p>ScripTree: scripting phylogenetic graphics Fran&ccedil;ois Chevenet, Olivier Croce, Maxime Hebrard, Richard Christen and Vincent Berry Bioinformatics 2010 26(8):1125-1126.</p>
	</li>
	<li><a href="http://paup.csit.fsu.edu/">PAUP*</a>: used for its MAST (Maximum Agreement SubTree) functionality&nbsp;and for computing most parsimonious&nbsp;trees in the MRP supertree method.
	<p>Swofford, D. L. 2003. PAUP*. Phylogenetic Analysis Using Parsimony (*and Other Methods). Version 4. Sinauer Associates, Sunderland, Massachusetts.</p>
	</li>
	<li><a href="http://evolution.genetics.washington.edu/phylip.html">PHYLIP package</a>: used for distance computation between two trees (<a href="http://evolution.genetics.washington.edu/phylip/doc/treedist.html">treedist</a> program with the &quot;symmetric difference&quot; option).
	<p>Felsenstein, J. 1993. PHYLIP (Phylogeny Inference Package) version 3.5c. Distributed by the author. Department of Genetics, University of Washington, Seattle.</p>
	</li>
	<li>
	<p><a href="http://www.cs.utexas.edu/~phylo/software/spruce/">Spruce</a> Python library: proposed by Rahul Suri and Tandy Warnow, making use of&nbsp;<a href="http://www.daimi.au.dk/~mailund/">Thomas Mailund</a>&#39;s open source&nbsp;<a href="http://www.daimi.au.dk/~mailund/newick.html">newick parser</a>.&nbsp;</p>
	</li>
</ul>

<h4>Links</h4>

<hr />
<p>Finding phylogenies and taxonomic information</p>

<ul>
	<li><a href="http://www.ncbi.orthomam.univ-montp2.fr/phyloexplorer/">PhyloExplorer</a>: check phylogenetic trees, get statistics, and many other functionnalities.</li>
	<li><a href="http://treebase.org/">TreeBASE</a>: a relational database designed to manage and explore information on phylogenetic relationships. This database can also be explored with Rod Page's <a href="http://iphylo.org/~rpage/phyloinformatics/treebase/">browser</a>.</li>
	<li><a href="http://tolweb.org/">Tree of Life</a>: navigate and retrieve information from the tree of life.</li>
	<li><a href="http://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/">NCBI taxonomy</a>: taxonomy resources, statistics, etc.</li>
</ul>
